src/controls/ControlPanelComboBox.js

```diff
@@ -370,7 +370,7 @@ class ControlPanelComboBox extends React.Component {
         </React.Fragment>
       )
     } else {
-      const title = simplified && simplified(label)
+      const title = simplified && simplified(label) || label
```

Comment:
I actually used here
const title = simplified && simplified(label) || undefined

If you set a title when the simplified value is not provided the combo box gets 2 entries, with the same value , for one ns ( one is bold - title, the other is regular , label )
If the title is undefined the combo looks like before

Do we want to have the old layout if no simplified function ?

Reply:
Oh, yes. I agree!